README.md

```diff
@@ -106,11 +111,11 @@ The `newState` method will be called by the `Store` whenever a new app state is
 
 Button taps result in dispatched actions that will be handled by the store and its reducers, resulting in a new app state.
 
-This is a very basic example that only shows a subset of Swift Flow's features, read the Getting Started Guide to see how you can build entire apps with this architecture.
+This is a very basic example that only shows a subset of ReSwift's features, read the Getting Started Guide to see how you can build entire apps with this architecture.
 
-[You can also watch this talk on the motivation behind Swift Flow](https://realm.io/news/benji-encz-unidirectional-data-flow-swift/).
+[You can also watch this talk on the motivation behind ReSwift](https://realm.io/news/benji-encz-unidirectional-data-flow-swift/).
```

Comment:
maybe make a note that it was Swift-Flow before hand? just for clarity for users not familiar with the framework already.